docs/developer-docs/latest/setup-deployment-guides/configurations/optional/functions.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Functions - Strapi Developer Documentation
+description: 
+---
+
+<!-- TODO: update SEO -->
+
+# Functions
+
+The `./config/functions/` folder contains a set of JavaScript files in order to add dynamic and logic based configurations.
+
+All functions that are exposed in this folder are accessible via `strapi.config.functions['fileName']();`
+
+<!-- The text above will be identified as a broken link by the check-links VuePress plugin, because its syntax looks like an empty link. You can safely ignore the error. -->
+
+## Bootstrap
```

Comment:
There is also a preboot file although I don't remember the exact name. Will check the RFCs to confirm later unless you have a doc from engineering that outlines it